py/server/deephaven/experimental/s3.py

```diff
@@ -23,6 +23,91 @@
     included in the package. This is an opt-out functionality included by default. If not included, importing this
     module will fail to find the java types.
 """
+
+class Credentials(JObjectWrapper):
```

Comment:
Ideally the pydocs should be self contained in some sense, potentially with references to other docs for details.  The docs for this class have way too few details.  E.g. I don't know if I'm setting env vars, properties, files in certain locations, etc.  That basically makes the docs and any code depending on this class useless.  Some of the links are nice and clear.  Others are not.

At a minimum, a user needs enough detail in the pydocs to know that they are in the right method, so it is worth taking the time to click on the link.